stubs/Access.stub

```diff
@@ -0,0 +1,15 @@
+<?php
+
+namespace App;
+
+use Illuminate\Database\Eloquent\Model;
+
+class Access extends Model
+{
+    public $timestamps = false;
+
+    protected $fillable = [
+        'netid', 'agent', 'path', 'created_at'
+    ];
```

Comment:
This is definitely a matter of opinion, but I'm a fan of just doing `protected $guarded = [];` to disable the fill protection. 

I find the fill protection more obnoxious than helpful after they made the Validator really awesome & strip out 'extra' undesirable fields people may maliciously submit.

Not a big deal here, just thought I'd share that in case you were interested :)